src/main/java/seedu/mark/model/bookmark/Folder.java

```diff
@@ -0,0 +1,55 @@
+package seedu.mark.model.bookmark;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.mark.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Folder in Mark.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidFolder(String)}
+ */
+public class Folder {
+    public static final String MESSAGE_CONSTRAINTS = "Folder names should be alphanumeric";
+    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+    public static final String DEFAULT_FOLDER_NAME = "ROOT";
+    public static final Folder ROOT_FOLDER = new Folder(DEFAULT_FOLDER_NAME);
```

Comment:
Just for consistency, maybe rename `ROOT_FOLDER` to `DEFAULT_FOLDER` or rename `DEFAULT_FOLDER_NAME` to `ROOT_FOLDER_NAME`;

Reply:
ok!

Reply:
Ill put this as an issue for now